index.js

```diff
@@ -9,10 +9,12 @@ const {catchError, filter} = require('rxjs/operators');
 const streamToObservable = require('@samverschueren/stream-to-observable');
 const readPkgUp = require('read-pkg-up');
 const hasYarn = require('has-yarn');
+const hostedGitInfo = require('hosted-git-info');
 const prerequisiteTasks = require('./lib/prerequisite');
 const gitTasks = require('./lib/git');
 const util = require('./lib/util');
 const publish = require('./lib/publish');
+const ghChecks = require('./lib/gh-checks');
```

Comment:
Let's use the full name in code: `ghChecks` => `githubChecks` and `gh-checks` => `github-checks`.

Reply:
I was a bit torn about that since `gh-got` is also abbreviated but, in general, I like it 👍 Changed in 7121ed0a8b50b4c434162eb2f8e904e166283407.